translations/README/HINDI.md

```diff
@@ -49,7 +49,7 @@
 
 | [अरबी](अनुवाद/README/ARABIC.md) | [फ्रेंच](अनुवाद/README/FRENCH.md) | [जर्मन](अनुवाद/रीडमे/जर्मन.एमडी) | [इतालवी](अनुवाद/रीडमे/इटालियन.एमडी) | [कोरियाई](अनुवाद/README/KOREAN.md) | [पोलिश](अनुवाद/README/POLISH.md) | [जापान](अनुवाद/README/JAPANESE.md) |
 | -------------------------------------------------------------- | ------------------------------------------------------- | ------------------------------------------------------- | ------------------------------------------------------- | ------------------------------------------------------ | ------------------------------------------------------ | ------------------------------------------------------ |
-| [पुर्तगाली](अनुवाद/रीडमी/पुर्तगाली.एमडी) | [सर्बियाई](अनुवाद/README/SERBIAN.md) | [स्पेनिश](अनुवाद/README/SPANISH.md) | [तुर्की](अनुवाद/README/TURKISH.md) | [बांग्ला](अनुवाद/README/BANGLA.md) | अंग्रेज़ी |
+| [पुर्तगाली](अनुवाद/रीडमी/पुर्तगाली.एमडी) | [सर्बियाई](अनुवाद/README/SERBIAN.md) | [स्पेनिश](अनुवाद/README/SPANISH.md) | [तुर्की](अनुवाद/README/TURKISH.md) | [बांग्ला](अनुवाद/README/BANGLA.md) | [अंग्रेजी](https://github.com/Syknapse/Contribute-To-This-Project/blob/master/README.md) |
```

Comment:
Those URL's ain't valid!

Reply:
i  was talking about other links on that table.

Reply:
Oh! Sorry! I will update it.

Reply:
Hey!, updated the broken links to other translations.